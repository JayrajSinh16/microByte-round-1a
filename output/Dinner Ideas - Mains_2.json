{
  "title": "Roast Chicken Ingredients:",
  "outline": [
    {
      "level": "H1",
      "text": "Shrimp and Grits Ingredients:",
      "page": 1
    },
    {
      "level": "H2",
      "text": "Spaghetti Carbonara Ingredients:",
      "page": 2
    },
    {
      "level": "H3",
      "text": "Stuffed Bell Peppers Ingredients:",
      "page": 2
    },
    {
      "level": "H3",
      "text": "Sweet and Sour Pork Ingredients:",
      "page": 3
    },
    {
      "level": "H3",
      "text": "Tandoori Chicken Ingredients:",
      "page": 3
    },
    {
      "level": "H3",
      "text": "Thai Green Curry Ingredients:",
      "page": 4
    },
    {
      "level": "H3",
      "text": "Turkey Meatballs Ingredients:",
      "page": 4
    },
    {
      "level": "H3",
      "text": "Vegetable Stir-Fry Ingredients:",
      "page": 5
    },
    {
      "level": "H3",
      "text": "Vietnamese Pho Ingredients:",
      "page": 5
    },
    {
      "level": "H3",
      "text": "Yakitori Ingredients:",
      "page": 6
    },
    {
      "level": "H3",
      "text": "Zucchini Noodles with Pesto Ingredients:",
      "page": 6
    },
    {
      "level": "H3",
      "text": "Beef and Broccoli Ingredients:",
      "page": 7
    },
    {
      "level": "H3",
      "text": "Chicken Fajitas Ingredients:",
      "page": 7
    },
    {
      "level": "H3",
      "text": "Chicken Piccata Ingredients:",
      "page": 8
    },
    {
      "level": "H3",
      "text": "Chicken Pot Pie Ingredients:",
      "page": 8
    },
    {
      "level": "H3",
      "text": "Chicken Stir-Fry Ingredients:",
      "page": 9
    },
    {
      "level": "H3",
      "text": "Ingredients:",
      "page": 10
    },
    {
      "level": "H3",
      "text": "Crab Cakes Ingredients:",
      "page": 10
    },
    {
      "level": "H3",
      "text": "Enchiladas Ingredients:",
      "page": 11
    },
    {
      "level": "H3",
      "text": "Fettuccine Alfredo Ingredients:",
      "page": 11
    },
    {
      "level": "H3",
      "text": "Fish and Chips Ingredients:",
      "page": 12
    },
    {
      "level": "H3",
      "text": "Lamb Chops with Mint Sauce Ingredients:",
      "page": 13
    },
    {
      "level": "H3",
      "text": "Lobster Tail Ingredients:",
      "page": 14
    },
    {
      "level": "H3",
      "text": "Osso Buco Ingredients:",
      "page": 14
    },
    {
      "level": "H3",
      "text": "Pad See Ew Ingredients:",
      "page": 15
    }
  ]
}