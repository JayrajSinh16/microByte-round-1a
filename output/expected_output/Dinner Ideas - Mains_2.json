{
    "title": "Dinner - Mains",
    "outline": [
        {
            "level": "H1",
            "text": "Roast Chicken",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 1
        },
        {
            "level": "H1",
            "text": "Shrimp and Grits",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Spaghetti Carbonara",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Stuffed Bell Peppers",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Sweet and Sour Pork",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Tandoori Chicken",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Thai Green Curry",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Turkey Meatballs",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Vegetable Stir-Fry",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Vietnamese Pho",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Yakitori",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Zucchini Noodles with Pesto",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Beef and Broccoli",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Chicken Fajitas",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Chicken Piccata",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Chicken Pot Pie",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Chicken Stir-Fry",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Coq au Vin",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Crab Cakes",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Enchiladas",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Fettuccine Alfredo",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Fish and Chips",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Grilled Cheese Sandwich",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Grilled Salmon",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Grilled Shrimp Skewers",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Lamb Chops with Mint Sauce",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Lobster Tail",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Osso Buco",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        },
        {
            "level": "H1",
            "text": "Pad See Ew",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        }
    ]
}